assets/thanos-querier/deployment.yaml

```diff
@@ -15,6 +15,20 @@ spec:
       labels:
         app.kubernetes.io/name: thanos-querier
     spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: In
+                  values:
+                  - thanos-querier
+              namespaces:
+              - monitoring
```

Comment:
I think this namespace needs to be changed @s-urbaniak I will remove this change from this PR, as I don't have the background behind the change. s

Reply:
Nvm it comes from kube-prometheus :D

Reply:
hmm .. the more i think about this the more i come to the conclusion that we will have to dependency-inject `openshift-monitoring` and `openshift-user-workload-monitoring` during runtime.